packages/vitest/src/node/logger.ts

```diff
@@ -262,7 +262,7 @@ export class Logger {
         + '\nThis might cause false positive tests. Resolve unhandled errors to make sure your tests are not affected.',
       ),
     )
-    this.error(c.red(divider(c.bold(c.inverse(' Unhandled Errors ')))))
+    this.error(errorBanner(' Unhandled Errors '))
```

Comment:
`errorBanner` already adds surrounding spaces, so this is now adding 2x spaces on both sides. Same thing in other `errorBanner` usage too.

Reply:
Oh it's actually visible on snapshots.